README.md

```
@@ -1,12 +1,37 @@
+- [Authentication](#authentication)
 - [Examples](#examples)
 - [Development](#development)
+- [Prerequisites](#prerequisites)
+- [Documentation](#documentation)
+- [Release Instructions](#release-instructions)
+- [License](#license)
+
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)